includes/posting.inc.php

```diff
@@ -1389,9 +1389,6 @@
      $smarty->assign('spam',intval($field['spam']));
      $smarty->assign('spam_check_status',intval($field['spam_check_status']));
      $smarty->assign('back',$back);
-     #$smarty->assign('page',$page);
```

Comment:
Ok, I will commit the PR. When I find additional places, I will open a new PR with new code changes.